src/index.ts

```diff
@@ -78,6 +78,7 @@ const getAssetFromKV = async (event: FetchEvent, options?: Partial<Options>): Pr
       ASSET_MANIFEST: __STATIC_CONTENT_MANIFEST,
       mapRequestToAsset: mapRequestToAsset,
       cacheControl: defaultCacheControl,
+      defaultMimeType: 'text/plain',
```

Comment:
```suggestion
      defaultMimeType: 'application/octet-stream',
```

LGTM, but I prefer use `application/octet-stream` as a default option.

Reply:
i think we'll want to keep `text/plain` as a default here as that's historically what it's been if a mime type cannot be found based on the file extension. don't feel the need for a breaking change here 😄

Reply:
yeah it's likely best to assume whatever was the default previously to avoid unintentional breaks.